tests/test_helpers.py

```diff
@@ -414,31 +414,27 @@ def test_set_content_disposition_bad_param() -> None:
 
 # --------------------- proxies_from_env ------------------------------
 
-def test_proxies_from_env_http(mocker) -> None:
+@pytest.mark.parametrize('protocol', ['http', 'https', 'ws', 'wss'])
+def test_proxies_from_env(mocker, protocol) -> None:
     url = URL('http://aiohttp.io/path')
-    mocker.patch.dict(os.environ, {'http_proxy': str(url)})
+    mocker.patch.dict(os.environ, {protocol + '_proxy': str(url)})
```

Comment:
Use monkeypatch here too:
```suggestion
def test_proxies_from_env(monkeypatch, protocol) -> None:
    url = URL('http://aiohttp.io/path')
    monkeypatch.setenv(protocol + '_proxy', str(url))
```